packages/hoppscotch-backend/src/infra-config/helper.ts

```diff
@@ -54,6 +64,125 @@ export async function loadInfraConfiguration() {
   }
 }
 
+/**
+ * Read the default values from .env file and return them as an array
+ * @returns Array of default infra configs
+ */
+export async function getDefaultInfraConfigs(): Promise<
+  { name: InfraConfigEnum; value: string }[]
+> {
+  const prisma = new PrismaService();
```

Comment:
I have a suggestion, rather than initialize a new instance of `PrismaService` here can't we just pass in the already initiated `PrismaService` from the file this function is being used in?

Reply:
One caller function (form helper.ts) `isInfraConfigTablePopulated()` can not provide prismaService object.